### PR TITLE
Create Controllers, Views and Routes 🚀

### PR DESCRIPTION
**Pull request by hsztan**
I create the user, users, user_post, and poser_posts routes to my project. I also add its respective controller and controller actions for the index and show its corresponding views. 😎

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -2,5 +2,9 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root "articles#index"
+  root "users#index"
+  get "/users/:user_id/posts" => "posts#index", as: :user_posts
+  get "/users/:user_id/posts/:id" => "posts#show", as: :user_post
+  get "/users" => "users#index", as: :users
+  get "/users/:id" => "users#show", as: :user
```
Comment (Deepakdanger) [inline config/routes.rb]: - [x] I think we can write more optimal code here. We don't need to define these routes like this. Rails already has helper called resources which we can use for this and expose only the routes we need with the only keyword, like so:
```ruby
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @hsztan, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (hsztan) [reply config/routes.rb]: It looks clean! Will do 💪

**Review by ShadyShawkat — APPROVED** [review]
## Hi @hsztan, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**